playbooks/network-ee-integration-tests/pre.yaml

```diff
@@ -60,8 +60,8 @@
         newdata:
           appliance:
             vars:
-              ansible_python_interpreter: /usr/bin/python3.8
-      when: container_image_version in ["stable-2.9", "stable-2.11"]
+              ansible_python_interpreter: "{{ test_ansible_python_interpreter }}"
+      when: container_image_version in ["stable-2.9", "stable-2.11"] and ansible_connection != "ssh" and test_ansible_python_interpreter is defined
```

Comment:
We should simplify this too:

```
when: test_ansible_python_interpreter is defined
```

which will break vyos integration tests, so we'll need to depends-on the ansible-zuul-jobs change here too.